docs/cli.rst

```diff
@@ -1,11 +1,265 @@
+
+* A description of what the task does
+* The specific Python class associated with this task
+* The syntax for running the command
+* Any options for the task
+
+Information on specific options includes:
+
+* The syntax for the option (``-o option_name value``).
+* If the option is required or optional.
+* A description of what the option does.
+
+Example output looks like this:
+
+.. code-block:: console
+
+    $ cci task info util_sleep
+    util_sleep
+
+    Description: Sleeps for N seconds
+
+    Class: cumulusci.tasks.util.Sleep
+
+    Command Syntax
+
+        $ cci task run util_sleep
+
+    Options
+
+        -o seconds SECONDS
+        Required
+        The number of seconds to sleep
+        Default: 5
+
+For additional information on flows use::
+
+    $ cci flow info <flow_name>
+
+where ``<flow_name>`` is the actual name of the flow.
+Information on specific flows includes:
+
+* A description of the flow.
+* The ordered steps (and substeps) of a flow.
+
+Example output looks like this:
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+
+
+Troubleshooting Errors
+----------------------
+Errors happen!
+That’s why our team strives to provide our users with options for efficiently working with them when they occur.
+If you need additional troubleshooting errors or stacktraces, you can reach out to our team on the `CumulusCI Trailblazer Community Group <https://trailblazers.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009M9Z>`_.
+
+
+
+Reporting Error Logs 
+^^^^^^^^^^^^^^^^^^^^
+Use the ``cci error gist`` command to send the most recent log file to a `GitHub gist <https://docs.github.com/en/github/writing-on-github/creating-gists>`_ so you can quickly and easily share logs with others. 
+
+For this feature to work you will need to ensure that your `github service is setup with the proper scopes <https://cumulusci.readthedocs.io/en/latest/tutorial.html#github-service>`_.
```

Comment:
We should change this to an internal link to the correct section of the getting started docs, because this tutorial.html URL will go away once the doc-reorg is merged